torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -347,7 +349,7 @@ void GpuLower::lower(Fusion* fusion, DataType index_type) {
 
   // Reorder expressions for loop-nest generation respecting computeAt
   // relationships
-  const auto exprs_sorted = reorderExprsForComputeAt();
```

Comment:
Why do we lose a `const`?

Reply:
Oops, that was necessary when I was playing a different approach. Will revert the change.